src/components/Notification/Notification.test.tsx

```diff
@@ -0,0 +1,168 @@
+import "jest-styled-components";
+import * as stories from "./Notification.stories";
+import React from "react";
+import ReactDOM from "react-dom";
+import { composeStories } from "@storybook/testing-react";
+
+const {
+	Regular, Standard, CustomIcon, Inactive
+} = composeStories(stories);
+
+describe("Notification - Standard - Active - Regular Text - Regular Icon", () => {
+	let container: HTMLDivElement;
+	const testId = "test-notification-id";
+	const messageId = "test-notification-message";
+    const titleId = "test-notification-title";
+    const closeId = "test-notification-x";
+    const title = "New Message"
+	const message = "Kresimir: Thank you for sharin.."
+
+	beforeEach(() => {
+		container = document.createElement("div");
+		document.body.appendChild(container);
+		ReactDOM.render(<Standard />, container);
+	});
+
+	afterEach(() => {
+		document.body.removeChild(container);
+		container.remove();
+	});
+
+	it("renders the component", () => {
+		const element = container.querySelector(`[data-testid="${testId}"]`);
+		expect(element).not.toBeNull();
+	});
+
+	it("renders title correctly", () => {
+		const element = container.querySelector(`[data-testid="${titleId}"]`);
+		expect(element?.textContent).toBe(title);
+	});
+
+    it("renders message correctly", () => {
+		const element = container.querySelector(`[data-testid="${messageId}"]`);
+		expect(element?.textContent).toBe(message);
+	});
+
+	it("should render left icon", () => {
+		const iconSVG = container.querySelector(`[data-testid="${testId}"] > svg`);
+		expect(iconSVG).not.toBeNull();
+	});
+
+    it("should render close icon", () => {
+		const iconSVG = container.querySelector(`[data-testid="${closeId}"] > svg`);
+		expect(iconSVG).not.toBeNull();
```

Comment:
this checks that there is an SVG but you can also check the textContent of the SVG and it will return its title text to make sure its the correct SVG, but that test is also in `<Icon />` so maybe its over kill 😅

Reply:
Will change test to match iconType with svg title